src/presenter/point-presenter.js

```diff
@@ -0,0 +1,112 @@
+import { render, replace } from '../framework/render.js';
+import EventsView from '../view/event-view.js';
+import EventEdit from '../view/event-edit.js';
+
+const PointMode = {
+  VIEW: 'view',
+  EDIT: 'edit',
+};
+
+export default class PointPresenter{
+
+  #eventsListView = null;
+  #point = null;
```

Comment:
Лучше переименовать, например так 
```suggestion
#eventsListView = null;

  #offersModel = null;
  #destinationsModel = null;

  #pointData = null;

  #pointViewComponent = null;
  #pointEditComponent = null;

  #handleEditModeChange = null;
  #handleDataChange = null;

  #mode = PointMode.VIEW;
```